kernel/dw1000.h

```diff
@@ -483,15 +484,17 @@ union dw1000_rx_time {
 #define DW1000_SAR_TEMP_MDEGC(sar, sar_23c) \
 	(23000 + ((100000 * ((sar) - (sar_23c)))) / 114)
 
-/* Maximum possible link quality indicator value */
+/* Maximum possible indicator values */
 #define DW1000_LQI_MAX 0xff
 
-/* Link quality indicator threshold
+/* Link quality indicator thresholds
  *
  * This is a somewhat arbitrary choice, providing a basic level of
  * protection against inaccurate receive timestamps.
  */
-#define DW1000_LQI_THRESHOLD_DEFAULT 16
+#define DW1000_SNR_THRESHOLD_DEFAULT	16
+#define DW1000_FPR_THRESHOLD_DEFAULT	64
+#define DW1000_NOISE_THRESHOLD_DEFAULT	256
```

Comment:
Just to double check: are these the values that you have found to work in practice?

Reply:
good defaults